packages/bygger/src/components/NavBar.jsx

```diff
@@ -1,65 +1,210 @@
 import { makeStyles } from "@material-ui/styles";
-import { navCssVariables, useAppConfig } from "@navikt/skjemadigitalisering-shared-components";
-import { Undertittel } from "nav-frontend-typografi";
+import { Edit, Eye, Globe, HomeFilled, People, Settings, System } from "@navikt/ds-icons";
+import { Dropdown, Header } from "@navikt/ds-react-internal";
+import { useAppConfig } from "@navikt/skjemadigitalisering-shared-components";
 import React from "react";
-import { Link } from "react-router-dom";
+import { Link, NavLink } from "react-router-dom";
 import { useAuth } from "../context/auth-context";
-import Row from "./layout/Row";
-import UserInfo from "./UserInfo";
 
 const useStyles = makeStyles({
   navBar: {
-    backgroundColor: navCssVariables.navDypBlaLighten40,
-    color: "#fff",
-    padding: "1.5rem 2rem",
+    height: "56px",
+    justifyContent: "center",
+    position: "relative",
+    "@media (max-width: 1040px)": {
+      justifyContent: "space-between",
+    },
   },
   navBarLocal: {
-    backgroundColor: navCssVariables.navOrange,
-    color: "#fff",
-    padding: "1.5rem 2rem",
+    backgroundColor: "#003260",
+    height: "56px",
+    justifyContent: "center",
+    position: "relative",
+    "@media (max-width: 1040px)": {
+      justifyContent: "space-between",
+    },
+  },
+  formsLink: {
```

Comment:
Flytt styling som ikke er i bruk her til FormMenu og FormNavigation. Vurder å splitt opp til egne komponenter på for f.eks NavLink